devtools/common/src/main/java/io/quarkus/maven/utilities/PomTransformer.java

```diff
@@ -266,25 +286,61 @@ public static Transformation addModule(String module) {
                         }
                     }
 
-                    final Text indent = document.createTextNode(indent1 + context.getIndentationString());
                     final Node moduleNode = document.createElement("module");
                     moduleNode.appendChild(document.createTextNode(module));
 
                     final NodeList modulesChildren = modules.getChildNodes();
                     final int len = modulesChildren.getLength();
                     Node ws;
                     if (len == 0 || (ws = modulesChildren.item(len - 1)).getNodeType() != Node.TEXT_NODE) {
-                        ws = document.createTextNode(indent1);
+                        ws = context.indent(1);
                         modules.appendChild(ws);
                     }
-                    modules.insertBefore(indent, ws);
+                    modules.insertBefore(context.indent(2), ws);
                     modules.insertBefore(moduleNode, ws);
                 } catch (XPathExpressionException | DOMException e) {
                     throw new RuntimeException(e);
                 }
             };
         }
 
+        public static Transformation addManagedDependency(String groupId, String artifactId, String version) {
+            return (Document document, TransformationContext context) -> {
+                try {
+                    Node dependencyManagementDeps = (Node) context.getXPath().evaluate(
+                            anyNs("project", "dependencyManagement", "dependencies"), document,
+                            XPathConstants.NODE);
+                    if (dependencyManagementDeps == null) {
+                        throw new IllegalStateException(
+                                String.format("//project/dependencyManagement/dependencies not found in [%s]",
+                                        context.getPomXmlPath()));
```

Comment:
You didn't want to add it in case it's missing?

Reply:
I think if it is not there it rather means that the user has misconfigured the path.

Reply:
Thinking of it again, let me add it.

Reply:
Solved in b990aa5